### PR TITLE
[Term Entry] Python Plotly - express .sunburst()

### LINKED ISSUE #5173: [Term Entry] Python Plotly - express .sunburst()
### Reason/inspiration (optional)

We would like a new term entry on the `.sunburst()` method of the [`express module`](https://www.codecademy.com/resources/docs/plotly/express) in Plotly, Python. The entry should go in a new file under `docs/content/plotly/concepts/express/terms/sunburst/sunburst.md`

The entry should include the following:

- A description of the term
- A ##Syntax section that includes the syntax of the method and its details
- An ##Example section that includes a ```py ``` codeblock and ```shell ``` that shows the example output.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by mamtawardhani**
### Description
added a new Term Entry for Python Plotly - express `.sunburst()`

### Issue Solved
Closes #5173 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/plotly/concepts/express/terms/sunburst/sunburst.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+- `names`: The column name, Series, or array-like object that provides the labels for the sectors in the sunburst chart. This defines what each circular sector will represent.
+- `parents`: The column name, Series, or array-like object that specifies the parent-child relationships in the hierarchy. The root node will have an empty string `""` as its parent.
+- `values`: The column name, Series, or array-like object used to determine the size of each sector. Larger values result in larger sectors in the chart.
+- `color`: Specifies the column name, Series, or array-like object whose values will be used to assign colors to the sectors. This is helpful for visual differentiation between groups.
+- `title`: An optional title for the sunburst chart. If not provided, no title is displayed.
+- `labels`: A dictionary that can be used to override the default labels for axis titles, legend entries, and hover information. This helps to customize the chart's display.
+
+> **Note:** The ellipsis in the syntax (...) indicates that there are additional optional parameters beyond those listed here to customize the 3D mesh.
```
Comment (PragatiVerma18) [inline content/plotly/concepts/express/terms/sunburst/sunburst.md:33]: The Note section mentions "customize the 3D mesh," which is irrelevant for this topic. This appears to be copied from documentation about 3D Plotly charts and should be removed or reworded to:

```suggestion
> **Note:** The ellipsis in the syntax (...) indicates that there are additional optional parameters beyond those listed here to customize the appearance and behavior of the sunburst chart.
```

File: content/plotly/concepts/express/terms/sunburst/sunburst.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+## Example
+
+Below is an example that demonstrates how to use the `.sunburst()` method with a hierarchical dataset in a Pandas DataFrame:
```
Comment (PragatiVerma18) [inline content/plotly/concepts/express/terms/sunburst/sunburst.md:37]: We should link Pandas here:

```suggestion
Below is an example that demonstrates how to use the `.sunburst()` method with a hierarchical dataset in a [Pandas](https://www.codecademy.com/resources/docs/pandas) `DataFrame`:
```

File: content/plotly/concepts/express/terms/sunburst/sunburst.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  - 'Graphs'
+  - 'Libraries'
+  - 'Plotly'
+  - 'Data'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.sunburst()`** method in the Plotly express module creates sunburst charts, which are circular charts that represent hierarchical data. The chart uses sectors to represent levels in the hierarchy, with the root level at the center and child levels radiating outward. Sunburst charts are particularly useful for visualizing nested data and their relationships.
```
Comment (PragatiVerma18) [inline content/plotly/concepts/express/terms/sunburst/sunburst.md:17]: We should link Plotly here:

```suggestion
The **`.sunburst()`** method in the [Plotly](https://www.codecademy.com/resources/docs/plotly) `express` module creates sunburst charts, which are circular charts that represent hierarchical data. The chart uses sectors to represent levels in the hierarchy, with the root level at the centre and child levels radiating outward. Sunburst charts are particularly useful for visualizing nested data and their relationships.
```

File: content/plotly/concepts/express/terms/sunburst/sunburst.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+# Customizing layout and margins
+fig.update_layout(margin=dict(t=50, l=25, r=25, b=25))
+
+# Displaying the sunburst plot
+fig.show()
+```
+
+The example above results in the following output:
+
+![The output for the example above](https://raw.githubusercontent.com/Codecademy/docs/main/media/sunburst-plotly.png)
```
Comment (PragatiVerma18) [inline content/plotly/concepts/express/terms/sunburst/sunburst.md:70]: The alt text for the image should be more descriptive:

```suggestion
![The output chart displays a circular hierarchy with the root at the centre and child nodes radiating outward. Each sector's size and color are proportional to its respective values and color attributes.](https://raw.githubusercontent.com/Codecademy/docs/main/media/sunburst-plotly.png)
```

**Review by PragatiVerma18 — CHANGES_REQUESTED** [review]
Hey @mamtawardhani, thanks for adding this entry. I have some suggestions, please check the comments and do the required changes. Let me know if you have any questions or need any help. Thanks.

**Comment by mamtawardhani** [comment]
Hey @PragatiVerma18, the changes are made. You can merge the PR. Thanks for the suggestions 😃

**Review by PragatiVerma18 — APPROVED** [review]

**Pull request merged**